README.md

```diff
@@ -82,11 +82,16 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: mynotifypod
-  # /var/lib/kubelet/seccomp/notify.json
+  # For older versions of Kubernetes:
```

Comment:
Thanks for the update!
For more context (could be added in the comment):
This annotation was deprecated in Kubernetes v1.19 and completely removed in v1.27

Reply:
Thanks